src/consensus/pbft/libbyz/prepared_cert.h

```diff
@@ -173,25 +177,30 @@ inline bool Prepared_cert::add(Prepare* m)
   return result;
 }
 
-inline bool Prepared_cert::add_mine(Prepare* m)
+inline bool Prepared_cert::add_mine(Prepare* m, uint64_t nonce)
```

Comment:
regarding `add_mine` for both `Pre_prepare` and `Prepare`, since both prepare and pre prepare contain the nonce, can't we use the messages (`m`) to get the nonce and set it here instead of passing it in to the `add_mine` method(s)?

Reply:
if you have a look at the pre-prepare and prepare they only have the hashed nonce not the unhashed nonce.
This is passing the unhashed nonce

Reply:
I ended up rewriting a bit this code is different and the comment no longer applies